hooks/submit_for_review.py

```diff
@@ -17,82 +17,65 @@
 
 class BasicSubmitForReviewPlugin(HookBaseClass):
     """
-    TODO: UPDATE TO SUBMIT FOR REVIEW!!!!!!
-
     Plugin for submitting a review in Shotgun.
 
-    This plugin is typically configured to act upon files that are dragged and
-    dropped into the publisher UI. It can also be used as a base class for
-    other file-based publish plugins as it contains standard operations for
-    validating and registering publishes with Shotgun.
+    This plugin is typically configured to act as a base class for
```

Comment:
I think we can get rid of the rest of the docstring from this point on.

Reply:
Removed